mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -85,6 +85,46 @@ CREATE TABLE mint_txs (
     FOREIGN KEY (mint_config_id) REFERENCES mint_configs(id)
 );
 
+-- Processed gnosis safe transactions
```

Comment:
They would've went in a separate migration if there were any users for this. Since it's currently not being used we get to do it all in one file, which makes it much easier for someone to look at/make edits.